rpcs3/Emu/Cell/PPULLVMRecompiler.cpp

```diff
@@ -7,7 +7,6 @@
 #include "llvm/Support/Host.h"
 #include "llvm/Support/ManagedStatic.h"
 #include "llvm/ExecutionEngine/GenericValue.h"
-#include "llvm/ExecutionEngine/MCJIT.h"

```

Comment:
But no, I deleted anything, looks better :)
I just deliver cleaner way as it was just before, that's all.